src/components/in-world-hud.js

```diff
@@ -53,12 +53,20 @@ AFRAME.registerComponent("in-world-hud", {
     this.onInviteClick = () => {
       this.el.emit("action_invite");
     };
+
+    this.onHubUpdated = () => {
+      if (window.APP.hub) {
```

Comment:
TIL we had a 3d invite button. Also I get why but this line reads as backwards if you don't realize we are now mixing what "invite" means.

Reply:
Decided not to refactor the existing code to rename the existing feature to "share", since it does look like we are going to rename things to "invite" in the new redesign. We'll have to revisit how these things work together at that time.